apps/contrib/translations.py

```diff
@@ -0,0 +1,33 @@
+from django.utils import translation
```

Comment:
nitpick: as we only use this in `cms`, should we just move it there instead of `contrib`?

Reply:
Or do we plan to ever use this somewhere else?

Reply:
yeah I was thinking of it

Reply:
Alright, then lets keep it in contrib